spec/system/consumer/shops_spec.rb

```diff
@@ -23,9 +23,38 @@
     producer.set_producer_property 'Organic', 'NASAA 12345'
   end
 
-  it "searches by URL" do
-    visit shops_path(anchor: "/?query=xyzzy")
-    expect(page).to have_content "Sorry, no results found for xyzzy"
+  context "searching enterprises" do
+    context "which exist" do
+      it "by URL" do
+        visit shops_path(anchor: "/?query=Enterprise")
+        expect(page).to have_content "Did you mean? #{distributor.name}"
+      end
+
+      it "by typing in the search field" do
+        visit shops_path
+        find('input').set("Enterprise")
```

Comment:
It's the only input on the page? This will probably break if we add another input on the page one day. It's much better to use `fill_in` but we would need a name for label for that. The headline should probably be a label...